.github/workflows/rails.yml

```diff
@@ -0,0 +1,45 @@
+name: Rails
+on: [push, pull_request]
+jobs:
+  Rails-Tests:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: rails_github_actions
+          POSTGRES_DB: rails_github_actions_test
+          POSTGRES_PASSWORD: postgres
+        ports: ["5432:5432"]
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+
+    - name: Setup test database
+      env:
+        RAILS_ENV: test
+        PGHOST: localhost
+        POSTGRES_DB: rails_github_actions_test
+        POSTGRES_USER: rails_github_actions
+        POSTGRES_PASSWORD: postgres
+      run: |
+        cp config/database.ci.yml config/database.yml
+        bundle exec rake db:create db:migrate
```

Comment:
is this required? db:migrate is executed when calling rails ci
see lib/tasks/ci.rake

Reply:
Yes, but the CI Task also tries to execute the frontend tests which are not intended to be used in this GitHub Action. We would probably have to split up the rake tasks if we wanted to use the ci rake task. 
I think having the commands in the workflow file makes it simpler to follow anyway.